pysmt/rewritings.py

```diff
@@ -673,6 +679,126 @@ def walk_minus(self, formula, args, **kwargs):
 # EOC TimesDistributivity
 
 
+
+class Ackermannization():
```

Comment:
Minor: Classes have names that are nouns that can be use as subject in a sentence (although sometimes this sounds weird, see CNFizer)